### PR TITLE
Added plugs `home` & `unity7`

### PR DESCRIPTION
**Pull request by soumyaDghosh**

### CODE REVIEW

**Comment by soumyaDghosh** [comment]
Seems like the snap is crashing, I don't know why and how

**Comment by nlogozzo** [comment]
No error messages?

**Comment by fsobolev** [comment]
home plug isn't needed since #223

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -136,3 +136,5 @@ apps:
       - network-status
       - removable-media
       - mount-observe
+      - home
```
Comment (nlogozzo) [inline snap/snapcraft.yaml:139]: Not needed

**Review by nlogozzo — CHANGES_REQUESTED** [review]

**Review by nlogozzo — APPROVED** [review]

**Pull request merged**